content/00.build/30.zksync-sso/00.index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Intro
+description: Learn about ZKsync Smart Sign-on.
+---
+
+ZKsync SSO is a smart wallet that simplifies user authentication, session management,
```

Comment:
Are we sure we want to stick to "smart wallet" phrasing? It's not just a wallet after all, it's a whole suite of things